src/main/java/org/springframework/integration/aws/inbound/SnsInboundChannelAdapter.java

```diff
@@ -42,6 +43,7 @@
 
 import com.amazonaws.services.sns.AmazonSNS;
 import com.fasterxml.jackson.databind.JsonNode;
+import com.sun.javafx.collections.ImmutableObservableList;
```

Comment:
JavaFX?
I don't think so. 
We really should try to stay with Java Core as much as possible.